docs/quickstart/index.rst

```diff
@@ -2,23 +2,40 @@
 Quickstart
 ==========
 
-Step 0: install package
+.. note::
+   These instructions describe a fully local workflow. To run on remote
+   resources, users may either `configure their resources themselves <https://qiskit-extensions.github.io/quantum-serverless/deployment/cloud.html>`_ or ask
+   their provider or dev ops team.
+
+Step 0: Close any running jupyter notebooks.
+
+Step 1: Close any running containers.
+.. code-block::
+   :caption: Clean up running containers.
+    
+      docker rm -f $(docker ps -aq)
```

Comment:
If the goal is to make sure the container is not _running_, I think this may be overkill (and potentially destructive).

Reply:
@IceKhan13 how stringent is the requirement the user kill all their containers before running demos?

At minimum, I can loosen up the language here to *suggest* they do this if they are having troubles? What specific troubles do we expect?

Reply:
I'm going to just remove this without adding a note for now.